### ISSUE TITLE
word2vec.py does not have a apply_gradients

### ISSUE THREAD

**Issue by LopezGG** [wangz10/tensorflow-playground#5]
the code does not appear to be connected on tensorboard.

**Comment by LopezGG** [comment]
you might want to chane the optimizer to include self.loss

```
# Optimizer.
				if self.optimize == 'Adagrad':
					self.optimizer = tf.train.AdagradOptimizer(self.learning_rate).minimize(self.loss)
				elif self.optimize == 'SGD':
					self.optimizer = tf.train.GradientDescentOptimizer(self.learning_rate).minimize(self.loss)
```
